### ISSUE TITLE
Android build failed execption

### ISSUE THREAD

**Issue by edgeadjei** [christopherdro/react-native-html-to-pdf#33]
The below exception was thrown during build for android everything works well in ios
```
/testApp/android/app/src/main/java/com/mosaic/MainApplication.java:6: error: cannot find symbol
import com.parkerdan.htmltopdf.RNHTMLtoPDFPackage;
                              ^
  symbol:   class RNHTMLtoPDFPackage
  location: package com.parkerdan.htmltopdf
/testApp/android/app/src/main/java/com/mosaic/MainApplication.java:30: error: cannot find symbol
            new RNHTMLtoPDFPackage(),
                ^
  symbol: class RNHTMLtoPDFPackage
2 errors
:app:compileDebugJavaWithJavac FAILED
```

**Comment by Tungnt2112** [comment]
I have the same issue! Please help!

**Comment by jraack** [comment]
Same issue here! Works great on IOS!

**Comment by DenizTemizer** [comment]
Open your MainApplication.java and look if the imported package is:
`import com.christopherdro.htmltopdf.RNHTMLtoPDFPackage;`

Somehow `react-native link` messed it up for me.

**Comment by christopherdro** [comment]
Thanks guys! Just released v0.4.0
